src/main/java/edu/gatech/cs2340/trydent/GameObject.java

```diff
@@ -25,7 +25,7 @@
 
     private String name = "Untitled GameObject";
 
-    private Group fxNode;
+    protected Group fxNode;
```

Comment:
If it must be accessible, I would prefer package visibility over protected, since I'm not sure even user-defined subclasses of GameObject should be able to set the underlying object. It could mess up the scene graph.